Multisig/App/RemoteNotificationHandler.swift

```diff
@@ -138,7 +146,9 @@ class RemoteNotificationHandler {
 
                 // At the time when permission granted, the token will be
                 // already set, so we need to register all stored safes
-                self.registerAll()
+                Task {
```

Comment:
Why we need a new task here  and on the other places?

Reply:
because we are getting delegate keys async now

Reply:
but we don't need to wait for the result, because we don't do anything even if the registration fails

Reply:
we are not waiting for the result. Reason for this task here is that we need means to invoke async function. Collecting delegate keys happens in async function now, which is called from other async function. And task here is for invoking async function from sync context.